core/src/main/java/org/jivesoftware/spark/ui/login/CertificatesManagerSettingsPanel.java

```diff
@@ -112,19 +114,30 @@ public Component prepareRenderer(TableCellRenderer renderer, int rowIndex,
 
 		ResourceUtils.resButton(acceptAll, Res.getString("checkbox.accept.all"));
 		ResourceUtils.resButton(acceptExpired, Res.getString("checkbox.accept.expired"));
-		ResourceUtils.resButton(acceptRevoked, Res.getString("checkbox.accept.invalid"));
+		ResourceUtils.resButton(acceptRevoked, Res.getString("checkbox.accept.revoked"));
 		ResourceUtils.resButton(acceptSelfSigned, Res.getString("checkbox.accept.self.signed"));
 		ResourceUtils.resButton(checkCRL, Res.getString("checkbox.check.crl"));
 		ResourceUtils.resButton(checkOCSP, Res.getString("checkbox.check.ocsp"));
 		ResourceUtils.resButton(showCert, Res.getString("button.show.certificate"));
 		ResourceUtils.resButton(fileButton, Res.getString("label.choose.file"));
-
+		
```

Comment:
Small bug: when I open the "certificates"  tab and the "accept all"  option is already checked, then the others are not disabled. It works when I unselect and reselect again.